google/cloud/automl/v1beta1/automl_gapic.yaml

```diff
@@ -623,6 +974,64 @@ interfaces:
       parent: location
     resource_name_treatment: STATIC_TYPES
     timeout_millis: 5000
+    samples:
+      standalone: 
+      - region_tag: automl_vision_list_models
+        value_sets: [automl_vision_list_models]
+    
+    sample_value_sets:
+    - id: automl_vision_list_models
+      description: Demonstrates using the AutoML client to get model details.
+
+      parameters:
+        defaults:
+        - parent%project="[Google Cloud Project ID]"
+        - parent%location="us-central1"
+        - filter="default_filter"
```

Comment:
Is `"default_filter"` a valid filter?

In the tests, you pass `""`. Should the default perhaps be an empty string?

I would try updating this to be an empty string :)